include/seqan3/search/configuration/all.hpp

```diff
@@ -60,6 +60,10 @@
  * | \ref search_configuration_subsection_hit_strategy "3. Hit"          |  ✅   |  ✅   |  ✅   |  ❌   |  ✅   |
  * | \ref seqan3::search_cfg::parallel "4: Parallel"                     |  ✅   |  ✅   |  ✅   |  ✅   |  ❌   |
  *
+ * \subsection search_configuration_search_result Search result type
+ *
+ * \copydetails seqan3::search_result
```

Comment:
is this not misplaced here? Should this not be part of the search/all.hpp?

Reply:
I thought it will help once the output configuration is also included but I can add it then.

Reply:
Mhmm I see. I just wondering if there was another documentation of the seach module.

Reply:
the search/all.hpp is very sparse and should proably be worked over anyway.